### PR TITLE
doc(lever): dbt model README edit for Fivetran style

### PR DESCRIPTION
**Pull request by fivetran-chloe**
Part of https://fivetran.height.app/T-59826

Edits for Fivetran style. Rewords for clarity. **Please check my edits for technical accuracy.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +69,6 @@ Please create issues or open PRs against `master`. Check out [this post](https:/
 
 ## Database Support
 This package has been tested on BigQuery, Snowflake and Redshift.
-Coming soon -- compatibility with Spark
```
Comment (fivetran-chloe) [inline README.md:74]: We do not advertise future changes - we only document them when they are live.

File: README.md [hunk 2]
```diff
@@ -1,26 +1,24 @@
...
-| [lever__requisition_enhanced](https://github.com/fivetran/dbt_lever/blob/master/models/lever__requisition_enhanced.sql)             | Each record represents a unique job requisition, enriched with information about the requisition's hiring manager, owner, any offers extended, and associated job postings. |
-| [lever__opportunity_stage_history](https://github.com/fivetran/dbt_lever/blob/master/models/lever__opportunity_stage_history.sql)             | Each record represents a stage that an opportunity has advanced to, with data regarding the time spent in each stage, the job's team, location, and department, the application source, the hiring manager, and the opportunity's owner.  |
+| [lever__interview_enhanced](https://github.com/fivetran/dbt_lever/blob/master/models/lever__interview_enhanced.sql)             | Each record represents a score that an interviewer gives to a unique interviewee. Includes data around the employees involved in this interview/opportunity, the interview feedback score standards, whether the opportunity advanced past this interview, how long the opportunity had been open at the time of the interview, and the opportunity source. |
```
Comment (fivetran-chloe) [inline README.md:17]: The abbreviation “i.e.” typically refers to a specific example, so I assume that “score” is the only piece of feedback that this table considers. Correct me if I’m wrong.

Comment (fivetran-chloe) [reply README.md:17]: Also, I assume “the employees involved” means the employees who are conducting the interviews (reviewers) and the individual feedback comes from them?

**Review by fivetran-jamie — APPROVED** [review]
thanks chloe!

**Pull request merged**